crates/router/src/core/payouts.rs

```diff
@@ -1899,12 +1967,24 @@ pub async fn create_recipient_disburse_account(
                 .attach_printable("Error updating payout_attempt in db")?;
         }
         Err(err) => {
+            let (error_code, error_message) = (Some(err.code), Some(err.message));
+            let (unified_code, unified_message) = helpers::get_gsm_record(
+                state,
+                error_code.clone(),
+                error_message.clone(),
+                payout_data.payout_attempt.connector.clone(),
+                consts::PAYOUT_FLOW_STR,
+            )
+            .await
+            .map_or((None, None), |gsm| (gsm.unified_code, gsm.unified_message));
             let updated_payout_attempt = storage::PayoutAttemptUpdate::StatusUpdate {
                 connector_payout_id: payout_data.payout_attempt.connector_payout_id.to_owned(),
                 status: storage_enums::PayoutStatus::Failed,
-                error_code: Some(err.code),
-                error_message: Some(err.message),
+                error_code,
+                error_message,
                 is_eligible: None,
+                unified_code,
+                unified_message,
```

Comment:
Since we are fetching it from gsm and not translating them, Will these unified code & message translated ?